tests/testMergeStructs.m

```diff
@@ -0,0 +1,21 @@
+classdef testMergeStructs < matlab.unittest.TestCase   
+
+    properties (TestParameter)
+        s = {struct('s1', 1, 's2', 2),...
+             struct('s1', 1),...
+             struct('s1', 1, 's2', 2, 's3', 3)};
+        d = {struct('d1', 1, 'd2', 2),...
+             struct('d1', 1),...
+             struct('d1', 1, 'd2', 2)};
+        r = {struct('s1', 1, 's2', 2, 'd1', 1, 'd2', 2),...
+             struct('s1', 1, 'd1', 1),...
+             struct('s1', 1, 's2', 2, 's3', 3, 'd1', 1, 'd2', 2)};
+    end
+
+    methods (Test, gitParameterCombination="sequential")
```

Comment:
Remove the word "git" here.

Reply:
Ohh It might have happened when I was committing it 😄